development/src/composables/videoTokenGenerator.js

```diff
@@ -0,0 +1,24 @@
+require('dotenv').config();
```

Comment:
is there a particular reason to use `dotenv`?

Reply:
Updated